core/src/main/groovy/com/novoda/gradle/release/PublishExtension.groovy

```diff
@@ -14,21 +16,33 @@ class PublishExtension {
      */
     @Deprecated
     String version
-    String publishVersion;
+
+    String publishVersion
 
     String[] licences = ['Apache-2.0']
 
     String uploadName = ''
+
     String description
     String website = ''
     String issueTracker = ''
     String repository = ''
     boolean autoPublish = true
+    String bintrayUser
 
-    String bintrayUser = ''
-    String bintrayKey = ''
+    String bintrayKey
     boolean dryRun = true
-
     String[] publications = ['maven']
 
+    private Project project
+
+    PublishExtension(Project project) {
+        this.project = project
+    }
+
+    @Override
+    Object getProperty(final String property) {
```

Comment:
Generated by AS. Should align with the general policy. 
Might move the `private` up to `project`. ;)